### PR TITLE
Initial package template

### PR DESCRIPTION
**Pull request by JohnMcIntyreJR**

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    person(given = "Jumping",
+           family = "Rivers",
+           role = c("aut", "cre"),
+           email = "training@jumpingrivers.com")
+Description: Practicals and data sets for the Machine Learning
+    with Tidymodels course run by Jumping Rivers.
+License: GPL-3
+URL: https://github.com/jr-packages/jrTidymodels
+Depends:
+    R (>= 3.2.0)
```
Comment (csgillespie) [inline DESCRIPTION:15]: R 3.5.0 would be better.

File: DESCRIPTION [hunk 2]
```diff
@@ -0,0 +1,33 @@
+Type: Package
+Package: jrTidymodels
+Title: Jumping Rivers: Machine Learning with Tidymodels
+Version: 0.0.1
+Authors@R: 
+    person(given = "Jumping",
+           family = "Rivers",
+           role = c("aut", "cre"),
+           email = "training@jumpingrivers.com")
```
Comment (csgillespie) [inline DESCRIPTION:9]: info@jumpingrivers.com - I think that's what we use elsewhere

File: .lintr [hunk 3]
```diff
@@ -0,0 +1,5 @@
+linters: with_defaults(
+    assignment_linter = NULL,
+    object_name_linter = NULL,
+    trailing_whitespace_linter = NULL
```
Comment (csgillespie) [inline .lintr:4]: Can you remove trailing_whitespace_linter

Also, do we need object_name as an exception

**Review by csgillespie — CHANGES_REQUESTED** [review]

File: .lintr (hunk 3 above)
Comment (JohnMcIntyreJR) [reply .lintr:4]: I agree trailing_whitespace_linter should be removed.  Do you mean that for object_name edge cases, we should use #nolint?

Comment (csgillespie) [reply .lintr:4]: Yes.

**Pull request merged**